AIR_Wheelly_API/Controllers/AuthController.cs

```diff
@@ -88,5 +90,25 @@ public async Task<IActionResult> TokenLogin(OAuthLoginDTO dto)
             var token = _authService.GenerateJwtToken(user.Id);
             return Ok(new { Token = token });
         }
+
+        [HttpPut]
+        public async Task<IActionResult> UpdateProfile([FromHeader] string authorization,
+            [FromBody] UpdateProfileDTO dto)
+        {
+            try
+            {
+                var jwtToken = authorization?.Replace("Bearer ", "");
```

Comment:
cisto fyi , [Authorize] anotacija to radi po defaultu ak se ne varam pa mozemo to samo koristiti, al i ovak je ok